infrastructure/main.tf

```diff
@@ -224,10 +181,7 @@ resource "aws_security_group" "workflows_api_handler_sg" {
   lifecycle { create_before_destroy = true }
 }
 
-# Function to build the JWKS URL
-locals {
-  build_jwks_url = "${format("https://cognito-idp.%s.amazonaws.com/%s/.well-known/jwks.json", local.aws_region, var.userpool_id)}"
-}
```

Comment:
Is this not used anywhere? that's why it was removed?

Reply:
https://github.com/NASA-IMPACT/veda-data-airflow/pull/156#discussion_r1613733005